requirements.txt

```diff
@@ -15,3 +15,5 @@ langdetect
 pyVim
 pyVmomi
 pytest
+requests
+pykwalify
```

Comment:
add the following please:
mock
git+https://github.com/demisto/demisto-py.git@test-content
slackclient